packages/material-ui-lab/src/SliderStyled/SliderStyled.js

```diff
@@ -303,6 +303,10 @@ const Slider = React.forwardRef(function Slider(inputProps, ref) {
   );
 });
 
+Slider.options = {
```

Comment:
This should be part of the JSON. If it's needed at runtime could you explain for what it is used? Preferably as a test.

Reply:
For the `styled()` components it is used only for the docs at this moment, so will add it as part of the JSON

Reply:
Restructured the data to contain both the name and the css info. No runtime changes are added at this moment.